CHANGELOG.md

```diff
@@ -22,7 +22,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added Databricks/Spark setup to the image. See README for setup & usage instructions.
 - Added rollback feature to `Databricks` source.
 - Changed all Prefect logging instances in the `sources` directory to native Python logging.
-- Changed `rm()`, `from_df()`, `to_df()` methods in `S3` Source
+- Changed `rm()`, `from_df()`, `to_df()` methods in `S3` source
+- Added `_exclude_whitespaces()` method in `SAPRFC` source
```

Comment:
Internal helper functions are normally not mentioned in changelog -- maybe you should read some changelogs of some public libraries to get a feel for what's typically included and how?